Mathlib/RingTheory/Nilpotent/Exp.lean

```diff
@@ -207,16 +207,16 @@ theorem exp_mul_of_derivation (R B : Type*) [CommRing R] [NonUnitalNonAssocRing
   have h_nilL : IsNilpotent DL := h_nil.map <| lTensorAlgHom R B B
   have h_nilR : IsNilpotent DR := h_nil.map <| rTensorAlgHom R B B
   have h_comm : Commute DL DR := by ext; simp [DL, DR]
-  let m : B ⊗[R] B →ₗ[R] B := lift <| LinearMap.mul R B
+  let m : B ⊗[R] B →ₗ[R] B := LinearMap.mul' R B
   have hm (x y : B) : m (x ⊗ₜ[R] y) = x * y := rfl
```

Comment:
Could drop this now in favor of `LinearMap.mul'_apply`

Reply:
OK done!